src/applications/disability-benefits/526EZ/components/ButtonContainer.jsx

```diff
@@ -66,7 +67,13 @@ export default function ButtonContainer(props) {
         </a>
       )}
       {!atGuidance() && (
-        <a className="usa-button-primary" onClick={goForward}>
+        <a
```

Comment:
Could we change this from an `a` to a `button`? That seems like a simpler approach.